src/Symfony/Component/JsonPath/JsonPathUtils.php

```diff
@@ -108,8 +114,8 @@ public static function unescapeString(string $str, string $quoteChar): string
                     "'" => "'",
                     '\\' => '\\',
                     '/' => '/',
-                    'b' => "\b",
-                    'f' => "\f",
+                    'b' => "\x08",
+                    'f' => "\x0C",
```

Comment:
why changing those ?

Reply:
`\f` is a mistake, but `\b` is actually not recognized by PHP, thus using the hex value directly